File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.py
class Solution:
    def subtractProductAndSum(self, n: int):
        n = list(str(n))
        res_m = 1
        res_s = 0
        for i in n:
            i = int(i)
            res_m *= i
            res_s += i
        return res_m - res_s
            
    
        
            

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.go
func subtractProductAndSum(n int) int{
    res_m := 1
    res_s := 0
    tmp := 0
    for n>0{
        tmp = n%10
        res_m *= tmp
        res_s += tmp
        
        n = n/10
    }
    
    
    return res_m - res_s
}